Configuration/Geometry/README.md

```diff
@@ -182,4 +183,5 @@ Several detector combinations have been generated:
 * D114 = T39+C19+M11+I17+O9+F8
 * D115 = T35+C20+M11+I17+O9+F8 
 * D116 = T35+C25+M12+I17+O10+F9
+* D117 = T35+C25+M12+I18+O10+F9
```

Comment:
("O10","T35","C25","M15","F9","I18") : "D117",

I think we should fix MXX for the whole readme, to avoid confusion with dictRun4Geometry.py

FYI @cms-sw/geometry-l2

Reply:
I fully agree, but given the large amount of work in this PR, and the time it is taking to converge, please let's fix this trivial, although important, documentation issue separately.